applications/fluid_structure_interaction/perpendicular_flap/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+IF(${EXADG_WITH_PRECICE})
```

Comment:
due to PR https://github.com/exadg/exadg/pull/192, ou also need to include the ExaDG header here (just copy it from another example).